Lib/os.py

```diff
@@ -36,7 +36,9 @@
 __all__ = ["altsep", "curdir", "pardir", "sep", "pathsep", "linesep",
            "defpath", "name", "path", "devnull", "SEEK_SET", "SEEK_CUR",
            "SEEK_END", "fsencode", "fsdecode", "get_exec_path", "fdopen",
-           "popen", "extsep"]
+           "extsep"]
+
+VXWORKS = (sys.platform == 'vxworks')
```

Comment:
This variable is only at a single place: you can remove it.

Reply:
removed.